src/shared/model/MutationalSignature.ts

```diff
@@ -0,0 +1,8 @@
+export interface IMutationalSignature {
+    sampleId: string;
+    patientId: string;
+    studyId: string;
+    mutationalSignatureId: string;
+    value: string;
```

Comment:
why isnt `value` a number?

Reply:
The mock data is string at this moment. @jxu8 we should define the API soon.